layouts/robots.txt

```diff
@@ -6,4 +6,10 @@ Disallow: /v1.1/
 Disallow: /404/
 Disallow: /404.html
 
+Allow: /docs/reference/generated/kubernetes-api/
```

Comment:
is this line the same (save for the disallow) as line 13?

Reply:
Oops. Good spot.

Reply:
The disallow was originally `Disallow: /docs/reference/generated/kubernetes-api/*` and I amended it.